### DISCUSSION TITLE
Split a stereo(2 channels in) audio into 2 mono audios by JavaCV

### DISCUSSION THREAD

**Discussion by steeveen** [bytedeco/javacv#2036] [Show and tell]
Hello everyone, I find a way to split a 2channel stereo audio into 2 single-channel mono audios just by using one FFmpegFrameGrabber and two FFmpegFrameRecorders(avoid using ProcessBuilder). And I have tested it on the 's16le' audio. Is this requirement universally useful？ Can we consider adding this feature to JavaCV？

**Comment by saudet** [comment]
It sounds like a good candidate to add to the samples directory:
https://github.com/bytedeco/javacv/tree/master/samples

What do you think?

**Reply by steeveen** [reply]
Hi. I clone the code from master branch. But the idea tell me "Cannot resolve org.bytedeco:opencv:4.7.0-1.5.10-SNAPSHOT". Should I begin my commit from 1.5.9?

**Reply by saudet** [reply]
Please try again with the snapshots: http://bytedeco.org/builds/

**Comment by steeveen** [comment]
Good.
I will complete it in my free time

**Comment by steeveen** [comment]
Hello, I have commited it.
